specification/sigma-appendix-modifiers.md

```diff
@@ -0,0 +1,148 @@
+
+- `contains`: Puts `*` wildcards around the values, such that the value is matched anywhere in the
+  field.
+
+* `exists`: Defines that a certain field has to exist or must not exist in a log event by providing a boolean value. Note that this check only verifies the presence of a field, not its value, be it empty or null.
+* `cased`: Values are applied case sensitively. Default Sigma behavior is case-insensitive matching.
+* `neq`: The field is different from the value. It's can be used with string or number.
+
+### String Modifiers
+
+The modifiers listed in this section can only be applied to string values.
+
+- `windash`: Creates all possible permutations of the `-`, `/`, `–` (en dash), `—` (em dash), and `―` (horizontal bar) characters. Windows command line flags can often be indicated by both characters. Using the `windash` modifier converts the aforementioned characters interchangeably and uses all possible permutation of strings in the selection.
+
+#### Regular Expression
+
+- `re`: Value is handled as a regular expression by backends. Regex is matched case-sensitive by default.
+
+  - Currently, the supported flavor is PCRE with the following metacharacters:
+    - Wildcards: `.`.
+    - Anchors: `^`, `$`.
+    - Quantifiers: `*`, `+`, `?`, `{n,m}`.
+    - Character Classes: [a-z], [^a-z].
+    - Grouping and Capturing: `()`.
+    - Alternation: `|`.
+  - The following metacharacters are unsupported:
+    - Character Classes: `[[:digit:]]`
+    - Lookahead Assertions:
+      - Positive Lookahead: `(?=...)`
+      - Negative Lookahead: `(?!...)`
+      - Positive Lookbehind: `(?<=...)`
+      - Negative Lookbehind: `(?<!...)`
+    - Atomic Grouping: `(?>`
+
+- `re` sub-modifiers:
+
+  - `i`: (insensitive) to enable case-insensitive matching.
+  - `m`: (multi line) to match across multiple lines. `^` /`$` match the start/end of line.
+  - `s`: (single line) to enable that dot (`.`) matches all characters, including the newline character.
+
+#### Encoding
+
+- `base64`: The value is encoded with Base64.
+
+- `base64offset`: If a value might appear somewhere in a base64-encoded string the representation
+  might change depending on the position of the value in the overall string. There are three variants for shifts
+  by zero to two bytes and except the first and last byte the encoded values have a static part in
+  the middle that can be recognized.
+
+- `base64` sub-modifiers:
+
+  - `utf16le`: Transforms value to UTF16-LE encoding, e.g. `cmd` > `63 00 6d 00 64 00`
+  - `utf16be`: Transforms value to UTF16-BE encoding, e.g. `cmd` > `00 63 00 6d 00 64`
+  - `utf16`: Prepends a [byte order mark](https://en.wikipedia.org/wiki/Byte_order_mark) and encodes UTF16, e.g. `cmd` > `FF FE 63 00 6d 00 64 00`
+  - `wide`: an alias for the `utf16le` modifier.
+
+### Numeric Modifiers
+
+The modifiers listed in this section can only be applied to numeric values.
```

Comment:
Should we mention `ne` here too? Someone who's interested in numeric comparison possibly doesn't reads the part above mentioning `neq`.